EnglishFilter/sections/general_extensions.txt

```diff
@@ -3280,6 +3280,8 @@ any.gs#%#var parts = document.URL.split("/url/"); if (parts.length == 2 && parts
 pirateproxy.in,pirateproxy.be#%#window.open=function() {};
 ! Fixing zive.cz video ad
 zive.cz#%#AG_onLoad(function() { window.VIDEO_AD_ENABLED = false; } );
+! https://github.com/AdguardTeam/AdguardFilters/issues/95305
+zive.cz#%#//scriptlet('set-constant', 'st', 'noopFunc')
```

Comment:
Hi. Good rule, but wrong section.
Added to correct place
https://github.com/AdguardTeam/AdguardFilters/commit/40939e4c7eaa8ec45a2806fd81369d9910adecf5